action.yaml

```diff
@@ -95,20 +100,59 @@ runs:
 
         cat >>$GITHUB_ENV <<EOF
         GITHUB_TOKEN=${{ github.token }}
+        INPUT_GITHUB_TOKEN=$INPUT_GITHUB_TOKEN
+        INPUT_TRUNK_TOKEN=$INPUT_TRUNK_TOKEN
+        EOF
+
+        # Every inputs.field should be referenced as INPUT_FIELD later in the action. This allows
+        # the field to be set either as an argument to the github action or via inputs.json.
+        if [[ -n "${{ fromJSON(inputs.json).version }}" ]]; then
+
+          cat >>$GITHUB_ENV <<EOF
+        GITHUB_EVENT_PULL_REQUEST_BASE_REPO_OWNER=${{ fromJSON(inputs.json).pull_request.base.repo.owner.login }}
```

Comment:
Any necessary error handling here for partially malformed json?

Reply:
i wish there was something we could do, but nope